src/schedlib/policies/satp3.py

```diff
@@ -178,6 +175,7 @@ def make_blocks(master_file):
     "################### Detector Setup######################",
     "with disable_trace():",
     "    run.initialize()",
+    "pysmurfs = run.CLIENTS['smurf']",
```

Comment:
This gets defined already at the top of the schedule [in the preamble](https://github.com/simonsobs/scheduler/blob/87551ae23f84596ec6d81565549e83120e968bd2/src/schedlib/policies/sat.py#L123).

I jumped in to comment because I'm trying to eliminate the need for this direct access to the clients by adding functionality to sorunlib where needed. It'd be great to get a response to #110.